lesson06/assignment/index.html

```diff
@@ -1,15 +1,122 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link rel="stylesheet" href="css/style.css" />
-        <title>Assignment 06 - Bootstrap 4</title>
-    </head>
-    <body>
-        <main class="main">
-          
-        </main>
-    </body>
-</html>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/style.css" />
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-gtEjrD/SeCtmISkJkNUaaKMoLD0//ElJ19smozuHV6z3Iehds+3Ulb9Bn9Plx0x4" crossorigin="anonymous">
+    </script>
+    <script src="index.js"></script>
+    <title>Assignment 06 - Bootstrap 4</title>
+</head>
+
+<body>
+
+    <img class="w-100 p-3 d-block" style="height: 250px; background-color: black;" src="images/banner.png"
+        alt="Harry Potter logo">
+
+
+    <ul class="nav" style="background-color: rgb(121, 80, 27)">
+        <li class="nav-item"><a href="#" class="nav-link text-warning">Home</a></li>
+        <li class="nav-item"><a href="#" class="nav-link text-warning">Blog</a></li>
+        <li class="nav-item"><a href="#" class="nav-link text-warning">Shop</a></li>
+        <li class="nav-item"><a href="#" class="nav-link text-warning">Join Club</a></li>
+    </ul>
+
+    <main class="main" style="background-color: lightgoldenrodyellow">
+        <div class="text-center p-5">
+            <img src="images/trio.jpeg" alt="Picture of Harry, Hermione, and Ron">
+        </div>
+
+        <div class="text-center">
+            <h1>Join the Club!</h1>
+        </div>
+
+        <h3 class="ms-4">Fill Out the Information Below to Join</h3>
+        <div class="p-3">
+            <form name="form1">
```

Comment:
Check out some of the Bootstrap examples in the documentation, there's some classes that will help line up all your form and add some additional styling.